Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1532,4 +1536,103 @@ namespace Babylon
     {
         return Napi::Value::From(info.Env(), m_graphicsImpl.GetHardwareScalingLevel());
     }
+
+    Napi::Value NativeEngine::CreateImageBitmap(const Napi::CallbackInfo& info)
+    {
+        const auto env = info.Env();
```

Comment:
nit: we are trying to use initializer lists everywhere and also don't use auto unless it's blatantly obvious what the type is
```suggestion
        const Napi::Env env{info.Env()};
```